tex/main.tex

```diff
@@ -585,44 +608,62 @@ \subsubsection{MPC Tracking Performance}
 The tracking error we recorded for 35 test trajectories after training both the eDMD 
 and jDMD models with an increasing number of trajectories. Like the cartpole example,
 results in Figure \ref{fig:rex_planar_quadrotor_mpc_test_error} clearly show that jDMD
-produces a high-quality model for MPC trackign with extremely few samples, whereas eDMD
+produces a high-quality model for MPC tracking with extremely few samples, whereas eDMD
 ---even after many training samples---is never able to develop a model with consistent
-and similar performance. The lack of progress with increasing samples is likely a result
-of poor variety in the training data: the jDMD model is not receiving any new information
-from additional training trajectories, and eDMD is not able to capture enough data that
-can capture enough behaviors that lie in the testing space. This is once again highlighted
+and similar performance. This is once again highlighted
 in the results of Figure \ref{fig:rex_planar_quadrotor_mpc_error_by_training_window}, which
 show that eDMD is only able to successfully track trajectories before falling apart when the
 range of test trajectories exceeds that of the training data. Meanwhile, jDMD is able to consistently
 track the trajectory to the goal state, despite having greater tracking error in a certain range
 of trajectories---this further hints at eDMD overfitting on the training data.
 
 \begin{figure}[t]
-	\centering
-	\begin{subfigure}[t]{0.48\textwidth}
-		\raggedleft
-		\includegraphics[width=\textwidth, height=5cm]{rex_planar_quadrotor_mpc_test_error.tikz}
-		\caption{MPC tracking error vs training samples for the planar quadrotor. The proposed jDMD method again learns a model good enough with ~1000 samples while eDMD is never able to converge to a good-enough model.}
-		\label{fig:rex_planar_quadrotor_mpc_test_error}
-	\end{subfigure}
-	\hfill
-	\begin{subfigure}[t]{0.48\textwidth}
-		\raggedright
-		\includegraphics[width=\textwidth, height=5cm]{rex_planar_quadrotor_mpc_error_by_training_window.tikz}
-		\caption{MPC tracking error of quadrotor over varying range of distribution for sampling initial conditions. Drastically increasing error for eDMD suggests greater overfitting.}
-		\label{fig:rex_planar_quadrotor_mpc_error_by_training_window}
-	\end{subfigure}
+    \centering
+    \begin{subfigure}[t]{0.48\textwidth}
+        \raggedleft
+        \includegraphics[width=\textwidth, height=5cm]{rex_planar_quadrotor_mpc_test_error.tikz}
+        \caption{MPC tracking error vs training samples for the planar quadrotor. The proposed jDMD method again learns a model good enough with ~1000 samples while eDMD is never able to converge to a good-enough model.}
```

Comment:
I'm starting to feel that "number of training trajectories" is more intuitive than "number of training sample." I think it better communicates efficiency of the algorithm and demonstrates that the methods converge with just a handful of training examples.

Reply:
might need to ask Zac for a preference. I'm personally indifferent, but # of samples might appeal to the general learning pple slightly more

Reply:
h